instruments/tests/test_keithley/test_keithley485.py

```diff
@@ -148,5 +148,5 @@ def test_measure():
                 "NDCL-9.0000E+0"
             ]
     ) as inst:
-        assert inst.measure() == 1.2345 * u.nanoamp
-        assert inst.measure() == 1. * u.nanoamp
+        assert 1.2345 * u.nanoamp == inst.measure()
+        assert 1 * u.nanoamp == inst.measure()
```

Comment:
Sometimes you replace "number * unit" notation to `u.Quantity(...)`. Sometimes it is not replaced. This is one occurrence. Change all for consistency?

Reply:
I think we can think about that in future PRs. This one is big enough as is, and something like that would be better served progressively.

Reply:
Agreed